Source/Scene/PostProcessBloomStage.js

```diff
@@ -0,0 +1,144 @@
+define([
+        '../Core/Check',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Shaders/PostProcessFilters/BloomComposite',
+        '../Shaders/PostProcessFilters/ContrastBias',
+        './PostProcess',
+        './PostProcessBlurStage',
+        './PostProcessComposite'
+    ], function(
+        Check,
+        defineProperties,
+        destroyObject,
+        BloomComposite,
+        ContrastBias,
+        PostProcess,
+        PostProcessBlurStage,
+        PostProcessComposite) {
+    'use strict';
+
+    /**
+     * Post process stage for bloom. Implements {@link PostProcess}.
+     *
+     * @alias PostProcessBloomStage
+     * @constructor
+     *
+     * @private
+     */
+    function PostProcessBloomStage() {
+        this._name = 'czm_bloom';
+
+        this._contrastBias = new PostProcess({
+            name : 'czm_bloom_contrast_bias',
+            fragmentShader : ContrastBias,
+            uniformValues : {
+                contrast : 0.0,
+                brightness : 0.0
+            }
+        });
+        this._blur = new PostProcessBlurStage({
+            name : 'czm_bloom_blur'
+        });
+        var generateComposite = new PostProcessComposite({
+            name : 'czm_bloom_contrast_bias_bloom',
+            processes : [this._contrastBias, this._blur]
```

Comment:
Throughout, we generally make member properties write once in a constructor function, e.g., they are assigned to but not read.  Instead, if they need to be read, they are first assigned to a local.  I'll check if it is in the Coding Guide, but I think it is good style.

Reply:
Submitted #6139